week02/largest_number.cpp

```diff
@@ -0,0 +1,29 @@
+bool compare(int a, int b){
+        string str1, str2;
+        stringstream ss1,ss2;
+        ss1<<a;
+        ss1>>str1;
+        ss2<<b;
+        ss2>>str2;
+        return str1+str2 > str2+str1;
+    }
+class Solution {
+public:
+    string largestNumber(vector<int>& nums) {
+        string result;
+        sort(nums.begin(),nums.end(),compare);
```

Comment:
It would be better in terms of complexity if you created another vector of strings, converted all integers into strings and push them, and finally sort them. Number of conversion from int to str would be much less.